_data/team.yml

```diff
@@ -149,3 +149,41 @@ grantees:
   link: https://twitter.com/theStack
   image: assets/images/sebastian-falbesoner.jpg
   alt: Picture of Sebastian Falbesoner
+
+- name: "0xB10C"
+  description: >-
+    0xB10C's funding supports his current and upcoming Bitcoin network monitoring
+    efforts, his work on Bitcoin Core development and review, his educational
+    efforts maintaining bitcoin-dev.blog and his work on reorgs on Signet.
+  social: "@0xB10C"
+  link: https://twitter.com/0xB10C
+  image: assets/images/0xb10c.png
+  alt: Picture of 0xB10C
+
+- name: "Vincenzo Palazzo"
+  description: >-
+    Vincenzo's funding supports his improvements and support the C-Lightning implementation
+    with new features, review of pull requests as well as review of the Lightning specification.
```

Comment:
Maybe: "Vincenzo's funding supports his work adding new features to C-Lightning and reviewing others' pull requests, as well as his ongoing contributions to the Lightning specification."

Reply:
Took this suggestion.